spec/importer_spec.rb

```diff
@@ -0,0 +1,178 @@
+require 'spec_helper'
+require 'ronin/masscan/importer'
+require 'masscan/output_file'
+require 'ronin/db'
+
+RSpec.describe Ronin::Masscan::Importer do
+  let(:output_file) { instance_double('Masscan::OutputFile') }
```

Comment:
We can test the output file end-to-end. There are example output files in `spec/fixtures/` that can be opened with `Masscan::OutputFile.new(path)`.